modules/switches.py

```diff
@@ -0,0 +1,93 @@
+import random
+import modules
+
+
+class Switches(modules.Module):
+    identifiers = ['switches']
+    display_name = "Switches"
+    manual_name = "Switches"
+    help_text = "`{cmd} flip 1 5 3 4 2 5 2` or `{cmd} flip 1534252`. Partial solutions allowed."
+    module_score = 1
+
+    def __init__(self, bomb, ident):
+        super().__init__(bomb, ident)
+
+        self.invalidPositions = "11011/10100/10000/01101/01100/01000/00111/00101/00011/00001".split("/")
+
+        self.solution = "10000"
+        while self.solution in self.invalidPositions:
+            self.solution = ''.join([str(random.randint(0, 1)) for i in range(5)])
+
+        self.switches = "10000"
+        while self.switches in self.invalidPositions or self.switches == self.solution:
+            self.switches = ''.join([str(random.randint(0, 1)) for i in range(5)])
```

Comment:
The code here is quite similar to the one above. Maybe this could be a separate function? `random_switches_state(invalid_states: set)`?